log_drains/remove.go

```diff
@@ -6,17 +6,49 @@ import (
 	"gopkg.in/errgo.v1"
 )
 
-func Remove(app string, URL string) error {
+type RemoveAddonOpts struct {
+	AddonID string
+	OnlyApp bool
+	URL     string
+}
+
+func Remove(app string, opts RemoveAddonOpts) error {
 	c, err := config.ScalingoClient()
 	if err != nil {
 		return errgo.Notef(err, "fail to get Scalingo client to remove a log drain from the application")
 	}
 
-	err = c.LogDrainRemove(app, URL)
-	if err != nil {
-		return errgo.Notef(err, "fail to remove the log drain from the application")
+	if opts.AddonID != "" {
+		// addons only
+		err := c.LogDrainAddonRemove(app, opts.AddonID, opts.URL)
+		if err != nil {
+			return errgo.Notef(err, "fail to remove the log drain from the addon %s", opts.AddonID)
+		}
+		io.Status("The log drain:", opts.URL, "has been deleted from the addon", opts.AddonID)
+	} else {
```

Comment:
```suggestion
          return nil
	}
```

I would remove the `else` section to simplify the code